dart/lib/src/protocol/sentry_stack_frame.dart

```diff
@@ -0,0 +1,106 @@
+class SentryStackFrame {
+  static final SentryStackFrame asynchronousGapFrameJson = SentryStackFrame()
+    ..absPath = '<asynchronous suspension>';
+
+  List<String> preContext;
+  List<String> postContext;
+  Map<String, String> vars;
+  List<int> framesOmitted;
+  String filename;
+  String function;
+  String module;
+  String lineno;
```

Comment:
```suggestion
  String lineNo;
```

Reply:
stick to always `camelCase`, in Java not always is like this because we use reflection and Gson name strategy, but here we do serialization by hands, it's fine to use whatever is easier to read

Reply:
✅